### PR TITLE
ch02s05: Lab

### PR DESCRIPTION
**Pull request by jramcast**
File and notebooks for this lab are available at:
* https://github.com/RedHatQuickCourses/rhods-qc-apps/pull/2

### CODE REVIEW

File: modules/chapter2/pages/section4.adoc [hunk 1]
```diff
@@ -0,0 +1,141 @@
+= Assessment Lab: Working with Data Science Projects
+:navtitle: Lab
+
+In this exercise, you will practice the use of data science projects, permissions, workbenches, environment variables, and data connections.
+
+[INFO]
+====
+*Prerequisites*
+
+To perform this exercise, ensure that your platform meets the following conditions:
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:10]: optionally, the oc client is needed?

File: modules/chapter2/pages/section4.adoc [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+* An S3 bucket.
+* The `data-scientists` user group exists in OpenShift.
+* The `user4` and `user5` users exist in OpenShift and are assigned to the `data-scientists` group.
+
+The identity provided of your demo platform already includes these users.
+To create the group, log in as an administrator user in OpenShift and then apply the https://github.com/RedHatQuickCourses/rhods-qc-apps/blob/ch02s04-lab/1.intro/chapter2/lab/group.yaml file.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:17]: How about mentioning that you can also apply the YAML using the OCP web console YAML editor? Warn them that this YAML should be applied to the correct OCP project/namespace

File: modules/chapter2/pages/section4.adoc [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  - Use the `Standard Data Science` image.
+  - This workbench must be connected to the S3 bucket that is available in your demo platform.
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_connection.ipynb` notebook in this workbench.
+
+* Create a workbench in `intro-projects-b`.
+  - Select any image for this workbench.
+  - Pass your name in the `STUDENT_NAME` environment variable.
+  - Clone the https://github.com/RedHatQuickCourses/rhods-qc-apps repository in this workbench.
+  - Execute the `rhods-qc-apps/1.intro/chapter2/lab/test_env_vars.ipynb` notebook in this workbench.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:44]: What's in this file? tell them to inspect it?

File: modules/chapter2/pages/section4.adoc [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+e. Click btn:[Create workbench] to finish the creation of the workbench and the data connection.
+
+
+3. Verify the connection of the workbench in `intro-projects-a`.
+
+a. Open the workbench of the `intro-projects-a` project.
+
+b. Log in to JupyterLab and accept the access form.
+
+c. Clone the https://github.com/RedHatQuickCourses/rhods-intro repository.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:84]: Move to qc-apps later?

File: modules/chapter2/pages/section4.adoc [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+f. Click btn:[Create workbench] to finish the creation of the workbench and the data connection.
+
+
+5. Verify the variable injected in the workbench in `intro-projects-b`.
+
+a. Open the workbench of the `intro-projects-b` project.
+
+b. Log in to JupyterLab and accept the access form.
+
+c. Clone the https://github.com/RedHatQuickCourses/rhods-intro repository.
```
Comment (rsriniva) [inline modules/chapter2/pages/section4.adoc:113]: move to qc-apps repo?

**Review by rsriniva — APPROVED** [review]
Great work! One or two screenshots in the solution would be nice. Address comments and merge. no need for rev-review. We will discuss with Ted if solution needs to be a separate section

File: modules/chapter2/pages/section4.adoc (hunk 2 above)
Comment (jramcast) [reply modules/chapter2/pages/section4.adoc:17]: Afaik, Groups are not namespaced

File: modules/chapter2/pages/section4.adoc (hunk 4 above)
Comment (jramcast) [reply modules/chapter2/pages/section4.adoc:84]: This is already in https://github.com/RedHatQuickCourses/rhods-qc-apps/pull/2 

I will adjust the repo URL.

File: modules/chapter2/pages/section4.adoc (hunk 5 above)
Comment (jramcast) [reply modules/chapter2/pages/section4.adoc:113]: This is already in https://github.com/RedHatQuickCourses/rhods-qc-apps/pull/2 

I will adjust the repo URL.

**Pull request merged**